src/gocam/cli.py

```diff
@@ -476,5 +480,273 @@ def flatten_models(input_file, input_format, output_format, output_file, fields)
         click.echo(output_content)
 
 
+@cli.command("translate-collection")
+@click.option(
+    "--url",
+    default="http://current.geneontology.org/products/json/noctua-models-json.tgz",
+    help="URL to download the GO-CAM models tarball",
+    show_default=True,
+)
+@click.option(
+    "--formats",
+    multiple=True,
+    type=click.Choice(["networkx", "cx2"]),
+    default=["networkx", "cx2"],
+    help="Formats to translate models to (can specify multiple)",
+    show_default=True,
+)
```

Comment:
Nitpick: it feels a little wonky to me to have one single `formats` option but two separate `--output-*` options. Would separate `--cx2` and `--networkx` flags make more sense? Maybe 🤷🏻  

Super nitpick: if you keep the single `formats` option, the click [docs](https://docs.python.org/3/library/itertools.html#itertools.islice) imply that options with `multiple=True` should still have a singular name.

Reply:
fixed --format

how about if I change the params to include 1 output param, but use that as the directory for which I will create subdirectories (named based on the --format list).  e.g., `--output=/tmp` and `--format=[networkx,cx2]` then full output path for each will be `/tmp/networkx/` and `/tmp/cx2`